chart/flux/flux-config.yaml

```diff
@@ -0,0 +1,5 @@
+git-url: "git@github.com:weaveworks/flux-get-started"
```

Comment:
This should contain a version, shouldn't it?

In fact ... shouldn't the helm installation e2e test be failing because of this?

Reply:
This is in the chart, and should probably be excised from this PR entirely.
I think it won't affect the end-to-end test because there's nothing in the chart that includes this file. Since it doesn't get mounted into the container, it doesn't get checked for validity.

Reply:
Ah, I see

Reply:
Let's remove the chart changes from this PR and address it separately.